public/app/features/datasources/settings/ButtonRow.tsx

```diff
@@ -17,6 +17,9 @@ const ButtonRow: FC<Props> = ({ isReadOnly, onDelete, onSubmit, onTest }) => {
       <LinkButton variant="secondary" fill="outline" href={`${config.appSubUrl}/datasources`}>
```

Comment:
While you're working on this, our UX team requested we make this a solid button (`fill="solid"`). Could you also please make that change? Thanks

Reply:
sure